src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/SqlCustomTagStore.cs

```diff
@@ -115,7 +115,7 @@ public async Task<IReadOnlyList<CustomTagStoreEntry>> GetCustomTagsAsync(string
 
         private static AddCustomTagsInputTableTypeV1Row ToAddCustomTagsInputTableTypeV1Row(CustomTagEntry entry)
         {
-            return new AddCustomTagsInputTableTypeV1Row(entry.Path, entry.VR, (byte)entry.Level);
+            return new AddCustomTagsInputTableTypeV1Row(entry.Path, entry.VR, entry.PrivateCreator, (byte)entry.Level);
```

Comment:
We started this whole conversation because of the mapping you where doing from IndexTag to DicomTag, I did not see that method removed.

Reply:
I split the whole user story to 3  tasks, that method will be removed as I update STOW
![image](https://user-images.githubusercontent.com/67345894/111662982-9f901f00-87cd-11eb-84be-7dd615f1bba5.png)